src/services/reg.js

```diff
@@ -0,0 +1,22 @@
+import backendClient from './http';
+
+export const userRegister = async (userName, email, password) => {
+  try {
+    const response = await backendClient.post(
+      '/api/auth/local/register',
+      {
+        username: userName,
+        email: email,
+        password: password,
+      },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+        },
```

Comment:
Don't need to specify this every time, it's enough to have it specified one time in the `http` service

```suggestion
```